lib/foods.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+  fetchFoods();
+});
+
+function fetchFoods () {
+  $.ajax({
+    method: "GET",
+    url: "http://localhost:3000/api/foods"
+  })
+  .then(function( foods ) {
+    let foodString;
+    foods.forEach (function( food ){
+      foodString = JSON.stringify(food)
+      localStorage.setItem(`${food.id}`, foodString)
```

Comment:
@TheKaterTot wasn't sure if we wanted the key to be the id or something else, any thoughts?